app/uk/gov/hmrc/apiplatformevents/repository/NotificationsRepository.scala

```diff
@@ -27,12 +27,14 @@ import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
 import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
 
 import uk.gov.hmrc.apiplatformevents.models.Notification
+import uk.gov.hmrc.apiplatform.modules.events.applications.domain.models.EventId
 
 object NotificationsRepository {
   import play.api.libs.json.{OFormat, Json}
-  import uk.gov.hmrc.apiplatform.modules.events.applications.domain.services.CommonJsonFormatters._
 
-  implicit def localDateTimeFormats()                    = MongoJavatimeFormats.localDateTimeFormat
+  implicit val eventIdJf = Json.valueFormat[EventId]
+
+  implicit def localDateTimeFormats()                    = MongoJavatimeFormats.localDateTimeFormat // TODO - change to instant
```

Comment:
Is this TODO meant to be done in this PR?

Reply:
No - it's a reminder to change notifications to Instant at a later date - here's already a lot changing.